package.json

```diff
@@ -27,13 +27,12 @@
   "devDependencies": {
     "@types/jest": "^22",
     "express": "^4.16.4",
-    "jest": "^22",
-    "publish": "^0.6.0",
+    "jest": "^24.8.0",
```

Comment:
had the most vulnerabilities

Reply:
Are we not using publish anywhere?

Reply:
oh, I see, you're patching and then publishing

Reply:
^

Reply:
most of the publish libraries seemed to not be maintained, I figure that would be good enough. and If we need to major we can manually change the package.json in a PR.